app/views/users/index.html.erb

```diff
@@ -1 +1,10 @@
-<h1>This is a list of all the user in the app.</h1>
+<h1>Users</h1>
+
+<div id="users">
+<% @users.each do |user| %>
+    <%= render user %>
+    <p>
+      <%= link_to "Show this user", user %>
+    </p>
+  <% end %>
+</div>
```

Comment:
- [ ] Please your user index file should match the given `http://127.0.0.1:3000/` [wireframes](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=ABIFLA3SVPWT6MXQ3C2OGFLBHIHES)
**User Wireframe**
![user_index](https://user-images.githubusercontent.com/55596970/186894132-aefb646b-0864-40f8-a578-d0f8cd33a204.png)
**Your Styling**
![youUserDesign](https://user-images.githubusercontent.com/55596970/186894594-696b7493-3199-4ec3-a163-457bc1f5cd06.png)